packages/ndla-pager/src/Pager.tsx

```diff
@@ -23,7 +22,7 @@ const createQueryString = (obj: Query) =>
 
 const pageItemActiveStyle = css`
   border-top: 3px solid ${colors.brand.primary};
-  background-color: ${rgba(colors.brand.lighter, 0.5)};
+  background-color: 'rgba(222,235,246,0.5)';
```

Comment:
Går det ikkje an å bruke colors her?

Reply:
Ikke direkte. css sin `rgba` støtter ikke hex.

Reply:
Usikker på gevinsten ved å endre dette om det medfører at vi må droppe bruk av colors konstanter. Det vil gjøre ting _litt_ meir tungvindt å bruke. Kva er gevinsten på bundle størrelse her?

Reply:
Vi må absolutt ikke droppe bruk av colors. Går helt fint å skrive `background-color: ${colors.brand.primary}` for eksempel. Dette er det eneste tilfellet der det ikke går, og dette kunne nok enkelt ha blitt unngått dersom vi hadde gått for litt annen css. Jeg skrev den bare sånn fordi jeg ville ha lavest mulig sannsynlighet for å endre noe. [Gevinsten er grei](https://bundlephobia.com/package/polished@4.2.2), men fordelen er at vi ender opp med en mindre ting man trenger å sette seg inn i.

Reply:
Det var ved bruk av rgba at eg meinte vi må droppe colors for hardkoda verdier.
Det var forresten overraskende stor størrelse på polished, men sjølve størrelsen i vår bundle er kanskje mindre på grunn av tree-shaking?

Reply:
Mja, vi får nok en liten reduksjon i størrelse men er litt usikker på tree-shaking i akkurat dette tilfellet. Vi har en babel-plugin som egentlig skal fjerne all bruk av polished under transpilering, men i og med at vi brukte det på en dynamisk måte vil ikke det være mulig. Denne bruken av `rgba` samsvarer med bruk av `rgba` andre steder i frontend-packages, så jeg så ikke på det som en big deal å gjøre det likt her også. Kan derimot godt fjerne polished, eller eventuelt oppgradere det.

Reply:
Det er mange rgba som bruker hex her: https://github.com/NDLANO/frontend-packages/blob/master/packages/core/scss/global/settings/settings.colors.scss
Vil alle disse slutte å virke om vi fjerner polished?

Reply:
Nei, disse ordner sass for oss.